modules/features/discover/src/main/java/au/com/shiftyjelly/pocketcasts/discover/view/DiscoverAdapter.kt

```diff
@@ -136,10 +136,17 @@ internal class DiscoverAdapter(
     }
     var onChangeRegion: (() -> Unit)? = null
 
+    private var isDiscoverVisible = true
```

Comment:
I think that this property name is a bit misleading. Something like `isFeaturePageTrackingEnabled` is more explicit and accurate.

Reply:
I agree! I've pushed the changes